addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml

```diff
@@ -24,7 +24,7 @@
                     <span class="o_AttachmentViewer_nameText" t-esc="attachmentViewer.attachment.displayName"/>
                 </div>
                 <div class="o-autogrow"/>
-                <div class="o_AttachmentViewer_buttonDownload o_AttachmentViewer_headerItem o_AttachmentViewer_headerItemButton" t-on-click="_onClickDownload" role="button" title="Download">
+                <div class="o_AttachmentViewer_buttonDownload o_AttachmentViewer_headerItem o_AttachmentViewer_headerItemButton" t-on-click="attachmentViewer.onClickDownload()" role="button" title="Download">
```

Comment:
Comment on the right line but also `o_AttachmentViewer_buttonDownload o_AttachmentViewer_toolbarButton` is still using the now removed handler and it should also get a new one instead.